### DISCUSSION TITLE
Unable to use Osiris

### DISCUSSION THREAD

**Discussion by AdyStudios** [danielkrupinski/Osiris#4210] [Q&A] [answered]
Hello! I'm probobly being dumb, but I just can't get osiris to open. I'm using extreme injector with manual mapping, and it says injection succesful. In the game this icon appears, but I can't open the cheat menu with INSERT, I don't know what is going on. I tried changing my keyboard layout, but that did not help. Is there any way I can rebind that key?
Thanks.
![image](https://github.com/danielkrupinski/Osiris/assets/74962285/603edd96-0ea4-4854-b984-14730cddbcc0)

**Comment by AdyStudios** [comment]
I have also tried smokey and "CSGO2 Injector" as well, with the same result.

**Comment by AdyStudios** [comment]
Where do I acces the aimbot, skinchanger etc?

**Reply by AdyStudios** [reply]
or are these gone now? In csgo there was all that stuff

**Reply by Xsintashi** [reply, marked as answer]
That bug button is the only thing that osiris offers rn
